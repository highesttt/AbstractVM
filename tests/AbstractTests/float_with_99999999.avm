push float(99999999)
push float(100000000)
div
dump
exit