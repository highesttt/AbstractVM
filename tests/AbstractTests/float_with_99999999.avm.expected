Return: 0
1
